### ISSUE TITLE
Fengsha/Drag Partition

### ISSUE THREAD

**Issue by bbakernoaa** [noaa-oar-arl/UFS-Aerosol-Config#31]
## Case Details: 

This case looks at several different things. 

1) The formulation of the horizontal flux is has been malformed according to a paper by [Webb et al., 2020](https://reader.elsevier.com/reader/sd/pii/S1875963719301661?token=3107D1A358DA879709755588457CFBC1FCB438DCFEDE34FF67088F4F53E22D9240EF8E3597607F767AA745DF0CC14537&originRegion=us-east-1&originCreation=20211026181050). I tested this out using the default configuration with the P7.1 AEROSOL only (so this is using the `FV3_GFS_v16` physics suite).

2) A new drag partition has been developed based on the 2016 paper by Adrian Chappel and Nick Webb. The new inputs are created at a 3 km global resolution monthly average of the input soil parameters for use in both regional and global applications of the Fengsha dust emission scheme. Note that a little more development needs to be done to filter out snow/ice cover in the new drag partition in higher latitudes.  It isn't impacting the results but we are seeing some sources spit out some high latitude dust.  

Prigent et al. 2015 
<img width="1055" alt="image" src="https://user-images.githubusercontent.com/22104759/139334361-8684fbb2-7c7e-4e43-bdae-3bdd61b34b24.png">

Developmental Albedo Drag
<img width="1023" alt="image" src="https://user-images.githubusercontent.com/22104759/139334626-1b1db1e1-24e8-4d27-acf8-552117aa7280.png">

## Model Configuration Details 
- Resolution: C384
- Begin Date: July 1 2019
- End Date: Aug 31 2019
- Cold Start: True 
- Forecast length (FHMAX_GFS): 24 hours
- FV3 output: every 6 hours 

## Aerosol Model Configuration

- ANTHRO1 (surface emissions) : CEDS
- ANTHRO2 (elevated emissions) : CEDS
- SHIP :  CEDS
- Biomass Burning: QFED where GBBEPx isn't available 
- Aviation: HTAP 
- NH3 Emissions: CEDS
- Dust : Fengsha
- AOD output: 3 hours 

Model Runs and run output directories

base case yaml file: `/scratch2/NAGAPE/arl/Barry.Baker/gwp71dev/workflow/cases/aerosol_firex_forecast.yaml`

1) Base case (old drag partition and without the horizontal mass flux fix) 
`/scratch2/NCEPDEV/naqfc/Barry.Baker/COM/basecase`
2) flux fix with alpha = 8 
`/scratch1/NCEPDEV/rstprod/tmp/COM/dragfix_alpha8`
3) flux fix alpha = 6
`/scratch1/NCEPDEV/rstprod/tmp/COM/dragfix_alpha6`
4) new drag partition + flux fix + alpha=2
`/scratch1/NCEPDEV/rstprod/tmp/COM/newdrag_alpha2`
5) new drag partition + flux fix + alpha=4
`/scratch1/NCEPDEV/rstprod/tmp/COM/newdrag_alpha4`

**Comment by zhanglikate** [comment]
@bbakernoaa Nice job. Which option do you  recommend to use, I can try one TEST for ATom run. Also, I am wondering is the ozone adjustment has been included in your above runs? I am a little concern the impact. I am planning to do a ATOM testing by turning off the O3 adjustment and quantify the impact, will keep you updated.

**Comment by bbakernoaa** [comment]
Yes the ozone adjustment is included.  I did not modify the source code
except for the horizontal flux formulation.

This does require a code change.  My gut is saying the new drag + flux
fix + alpha=2 but I think we might want to get a clearer picture of it.  It
may be between 2-4.

**Comment by ghost** [comment]
@bbakernoaa I looked into your runs and here is comparison with AERONET sites (your last two runs are still ongoing) : 
https://docs.google.com/presentation/d/1FeCpomQp4ZQylzYIjz1TFqvWPrKtloQi/edit?usp=sharing&ouid=100966050662930876047&rtpof=true&sd=true

**Comment by bbakernoaa** [comment]
Thanks Partha

I’m very encouraged by these results. It confirms my intuition that then
ndrag_alpha2 is pretty close. Thanks again for plotting this up

Would you mind also comparing to MERRA2 or ICAP?

**Comment by ghost** [comment]
@bbakernoaa in slide 4 and 5 compared against ICAP and GEOS5 already there.

**Comment by bbakernoaa** [comment]
Sorry Partha. It confirms what I expected. The newdrag_alpha2 looks the
best here to me. It may even be able to back down the scaling factor a
little bit. It’s surprising to me that GEOS5 was so low for this even
compared to ICAP.

GEOS also seems to keep that high bias over the Taklamakan. It look much
better in ICAP.

The underproduction in the Middle East needs further investigation. It did
improve compared to GEOS5 but it still seems that there is under prediction
of the total AOD here. I think that at this point we need to look at the
surface winds to see if it is seasalt over the ocean too. I did turn down
the source over Somalia that would increase this but it is still active and
I’m hesitant to do more without more analysis.

**Comment by bbakernoaa** [comment]
@perthsb It looks like the run is still going as you said. The ndrag_alpha2 should be finished in the next 10 hours or so

**Comment by bbakernoaa** [comment]
@zhanglikate 

Could you send me the control file you did for atom? I'll start a run since this one requires a code change

**Comment by bbakernoaa** [comment]
@perthsb Just to show you the dust AOD from the 14th of august with the ndrag_alpha2 
<img width="1023" alt="image" src="https://user-images.githubusercontent.com/22104759/139449442-12857716-a011-4f70-ac97-c6d4aa515f0b.png">

You can see the active sources over the Middle East and Saudi Arabia as well as the Horn of Africa. I'm really wondering if this bias is real or not.  I can't imagine that the entire area is under such an under bias compared to observations.  Dust AODs over the Indian Ocean are about 0.2 0.3 on average in this image but the total AODs are approximately 0.8.  I'm actually questioning the GEOS5 and ICAP results. I'm just suggesting that we look further into it before we declare dust being the sole source.

**Comment by zhanglikate** [comment]
@bbakernoaa  What did you mean control file? This  is my job submit folder, I think you can find all of the files and information you need: /scratch2/BMC/gsd-fv3-dev/NCEPDEV/global/Kate.Zhang/fv3gfs/expdir/DEV_96_fire, let me know if you have any questions. 

Here is the workflow folder: /scratch2/BMC/gsd-fv3-dev/lzhang/aerosols-workflow/

**Comment by ghost** [comment]
@bbakernoaa In ICAP there is high AOD over gulf of Arabia before 14th and it may be moving over the Arabian Sea. As they do not provide Seasalt or other species I am not sure  about the contribution by others in total AOD. 
In GEOS5, the high AOD is less than ICAP over Arabian peninsula and adjoining ocean.

**Comment by ghost** [comment]
@bbakernoaa @zhanglikate I have updated the ppt I sent yesterday, updating AERONET plot with "newdrag_alfa2" now reached 3rd week of August. Also in slide 6 comparison with MERRA2 is added now.

**Comment by bbakernoaa** [comment]
Thanks Partha!

I got yelled at to clean up some space so I'm going to backup
newdrag_alpha2 and then delete the rest.  I think that it is the clear
winner here and I'm glad that this has been resolved.

**Comment by ghost** [comment]
@bbakernoaa also your /scratch1/NCEPDEV/rstprod/tmp/COM/dragfix_alpha6 run stopped at 6th August. 
But Newdrag Alpha 2 is promising among the 5 runs.

**Comment by ghost** [comment]
@zhanglikate run ATOM case (201608) with no adjustment of O3 and here is the comparison against run with adjustment of O3 : 
https://docs.google.com/presentation/d/1rmboDAafQDEuUJ8ZxPQvzLaqpakL9Nlu/edit?usp=sharing&ouid=100966050662930876047&rtpof=true&sd=true
